packages/web-components/src/utils/date-utils.ts

```diff
@@ -173,7 +173,8 @@ export function validate(
   // Empty fields are acceptable unless the component is marked as required
   if (year && (year < minYear || year > maxYear)) {
     component.invalidYear = true;
-    component.error = i18next.t('year-range', { start: minYear, end: maxYear });
+    component.error = 'year-range';
+    component.errorParameters = { start: minYear, end: maxYear };
```

Comment:
We currently do not have the functionality to allow `minYear` and `maxYear` to be controlled externally, and no team has requested additional flexibility around these values so I believe we don't need to. However, we could create a utility method that returns the `start` and `end` values based on the specific error. Instead of using the `errorParameter` prop, we could call this utility method when needed.

Reply:
When you add a new `@Prop` though, it is public. Maybe I did not entirely understand your approach or the requirements of the ticket but I don't think that we want to solve this by creating a new prop. It will appear in the documentation and [teams will be able to access it](https://1488-memorable-date-error-translate--60f9b557105290003b387cd5.chromatic.com/?path=/docs/components-va-memorable-date--default):

![Screenshot 2023-03-31 at 8 39 37 AM](https://user-images.githubusercontent.com/872479/229135746-1bd12d94-0de5-4899-b70b-6949488452c4.png)

Your idea of instead using a separate internal utility method instead of a prop might be a better approach to explore.